clients/admin-ui/src/types/api/models/StagedResourceTypeValue.ts

```diff
@@ -16,4 +16,5 @@ export enum StagedResourceTypeValue {
   IMAGE = "Image",
   I_FRAME = "iFrame",
   JAVASCRIPT_TAG = "Javascript tag",
+  OKTA_APP = "okta_app",
```

Comment:
It looks like the rest are Titlecase; snake_case is inconsistent. `"Okta App"` seems like it makes more sense